### DISCUSSION TITLE
ADC with DMA

### DISCUSSION THREAD

**Discussion by AirchipTeknoloji** [stm32duino/Arduino_Core_STM32#2311] [Q&A] [answered]
Hello,

I want to use STM32 adc pins with dma but couldn't find any library or example code working with Stm32 Arduino Core. Anybody has any suggestion? 

STM32F0x or STM32F1x will be used mostly.

Thanks in advance.

Aykut.

**Comment by fpistm** [comment, marked as answer]
Hi @AirchipTeknoloji 
You can find here an example of a STM32Cube IDE project converted to Arduino sketch:
https://www.stm32duino.com/viewtopic.php?t=110

**Comment by AirchipTeknoloji** [comment]
Hi @fpistm ,

Thanks for reply, it seems nice. I will try & let you know.
